Submitted By: Jim Gifford <jim at linuxfromscratch dot org>
Date: 2006-07-04
Initial Package Version: 0.26.1
Origin: Various
Description: Make neon-config Multilib Friendly

diff -Naur neon-0.26.1.orig/neon-config.in neon-0.26.1/neon-config.in
--- neon-0.26.1.orig/neon-config.in	2006-02-26 12:55:43.000000000 -0800
+++ neon-0.26.1/neon-config.in	2006-07-04 13:19:15.091798399 -0700
@@ -5,7 +5,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
-libdir=@libdir@
+
+libdir=`pkg-config --variable=libdir neon`
+CFLAGS=`pkg-config --cflags neon`
+LIBS=`pkg-config --libs neon`
+has_lfs=`pkg-config --variable=has_lfs neon`
 
 usage()
 {
@@ -23,7 +27,7 @@
   --support FEATURE     exit with success if feature is supported
 
  Known features: 
-    dav [@NE_FLAG_DAV@], ssl [@NE_FLAG_SSL@], zlib [@NE_FLAG_ZLIB@], ipv6 [@NE_FLAG_IPV6@], lfs [@NE_FLAG_LFS@],
+    dav [@NE_FLAG_DAV@], ssl [@NE_FLAG_SSL@], zlib [@NE_FLAG_ZLIB@], ipv6 [@NE_FLAG_IPV6@], lfs [@has_lfs@],
     i18n [@NE_FLAG_I18N@], ts_ssl [@NE_FLAG_TS_SSL@]
 
 EOF
@@ -69,11 +73,10 @@
 	;;
 
     --cflags)
-       	echo -I${includedir}/neon @NEON_CFLAGS@
+       	echo ${CFLAGS}
        	;;
 
     --libs)
-        LIBS="-lneon @NEON_LIBS@"
 	# Don't add standard library paths
         if test "$prefix" != "/usr"; then
            LIBS="-L${libdir} ${LIBS}"
@@ -93,7 +96,7 @@
 	zlib|ZLIB) support @NE_FLAG_ZLIB@ ;;
 	ipv6|IPV6) support @NE_FLAG_IPV6@ ;;
 	dav|DAV) support @NE_FLAG_DAV@ ;;
-	lfs|LFS) support @NE_FLAG_LFS@ ;;
+	lfs|LFS) support @${has_lfs}@ ;;
 	i18n|I18N) support @NE_FLAG_I18N@ ;;
 	ts_ssl|TS_SSL) support @NE_FLAG_TS_SSL@ ;;
 	*) support no ;;
diff -Naur neon-0.26.1.orig/neon.pc.in neon-0.26.1/neon.pc.in
--- neon-0.26.1.orig/neon.pc.in	2004-10-02 11:47:02.000000000 -0700
+++ neon-0.26.1/neon.pc.in	2006-07-04 13:17:36.872068504 -0700
@@ -2,6 +2,7 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
+has_lfs=@NE_FLAG_LFS@
 
 Name: neon
 Description: HTTP/WebDAV client library
